cypress/integration/page.spec.js

```diff
@@ -949,23 +1027,23 @@
     });
 
     it('should allow to enter an author name', () => {
-      newCommentForm.nameInput().type('Some name')
+      newCommentForm.nameInput().type('Some name');
```

Comment:
Consider using a more descriptive name for the test input to improve readability.